app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::UsersController < Api::BaseController
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(user_params)
+      render json: @user, serializer: UserIndexSerializer, status: 201
+    else
+      error_response object: @user, status: 401
```

Comment:
Let's invoke this with the new method `unprocessable_response` (that way you don't need to pass it a status code)!